src/components/UserCard/index.tsx

```diff
@@ -0,0 +1,85 @@
+import { Maybe } from "src/generated/graphql";
+
+import AvatarBase from "../Avatars/AvatarBase";
+import BoxBase from "../Boxs/BoxBase";
+import ButtonBase from "../Buttons/ButtonBase";
+import DividerBase from "../Dividers/DividerBase";
+import TypographyBase from "../Typography/TypographyBase";
+
+export interface IUser {
+    name?: string;
+    email?: string;
+    isActive?: boolean;
+    avatar?: Maybe<string>;
+    userId?: string;
+}
+export interface IUserProps extends IUser {
+    name: string;
+    email: string;
+    isActive: boolean;
+    avatar?: Maybe<string>;
+    onUpdate: (userId: string) => void;
+    onChangeStatus: (userId: string) => void;
+    userId: string;
+}
+
+export const anonymousAvatarLink =
+    "https://cdn1.vectorstock.com/i/thumb-large/22/05/male-profile-picture-vector-1862205.jpg";
+const UserCard = (userProps: IUserProps) => {
```

Comment:
I think we should put user's exclusive components in /pages/user or something

Reply:
![image](https://user-images.githubusercontent.com/58205909/131253166-05754884-f7e8-4906-a6e2-0806f6c076b0.png)

Reply:
@kien123456k hello please help me check again